templates/addHiringPartner.phtml

```diff
@@ -1,42 +1,52 @@
 <!doctype html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8">
-        <meta name="viewport"
-              content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-        <meta http-equiv="X-UA-Compatible" content="ie=edge">
-
-        <link rel="stylesheet" type="text/css" href="css/genericStyles.css"/>
-        <link rel="stylesheet" href="css/admin.css">
-        <link rel="stylesheet" href="css/addHiringPartner.css">
-
-        <title>AcademyPortal | Add Hiring Partner</title>
-    </head>
-    <body>
-        <div>
-            <a href="/admin" class="returnButton">Return to admin page</a>
-        </div>
-        <h1> Add Hiring Partner Page</h1>
-        <div id="message"></div>
-        <form id="addHiringPartnerForm">
+<head>
+    <title>AcademyPortal | Add Hiring Partner</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="css/genericStyles.css"/>
+    <link rel="stylesheet" type="text/css" href="css/admin.css">
+    <script src="js/addHiringPartner.js" defer></script>
+</head>
+<body>
+    <div>
+        <a href="/admin" class="returnButton">Return to admin page</a>
+    </div>
+    <h1>Add Hiring Partner Page</h1>
+    <div id="message"></div>
+    <form id="addHiringPartnerForm">
+        <div class="formField">
             <label for="companyName" class="hiringPartnerLabels">Company name</label>
-            <input type="text" id="companyName" class="addHiringPartner" name="companyName" required>
+            <input type="text" id="companyName" class="addHiringPartner field" name="companyName" data-required="true" required>
```

Comment:
I can submit a blank company name by submitting '  '. Same for the tech stack.

Reply:
This validation is to stop people forgetting to input values rather than enforcing correct values.